sql/moz-fx-data-shared-prod/fxci_derived/task_run_costs_v1/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+friendly_name: Firefox-CI Task Run Costs
+description: |-
+  Derived Firefox-CI task run cost data.
+owners:
+- ahalberstadt@mozilla.com
+labels:
+  incremental: true
+  owner1: ahalberstadt
+  dag: bqetl_internal_tooling_phase_2
+scheduling:
+  dag_name: bqetl_internal_tooling_phase_2
+  task_name: fxci_task_run_cost__v1
```

Comment:
```suggestion
  dag_name: bqetl_internal_tooling
```

it should be fine to add this to the existing DAG. Also no need to override the task name (unless there is a good reason for doing so)